src/App.jsx

```diff
@@ -1,7 +1,22 @@
-import "./App.css";
-
+import './App.css';
+import Header from './components/common/header/Header';
+import CategoryFilter from './components/main/categoryFilter/CategoryFilter';
+import RestaurantList from './components/main/restaurant/RestaurantList';
+import AddRestaurantModal from './components/aside/AddRestaurantModal';
+import RestaurantDetailModal from './components/aside/RestaurantDetailModal';
 function App() {
```

Comment:
```suggestion
import RestaurantDetailModal from './components/aside/RestaurantDetailModal';

function App() {
```
요렇게 간격이 있으면 보기 더 좋지 않을까요??